src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -59,6 +59,7 @@ def create(data,
         # Make predictions on the training set and as column to the SFrame
         >>> test_data['predictions'] = model.predict(test_data)
     """
+    _tkutl._raise_error_if_column_exists(data, target, "dataset", target)
```

Comment:
The variable name is `data` so the third argument, `dataset_variable_name`, should be `"data"`